packages/lab/src/list-next/ListNext.tsx

```diff
@@ -22,11 +22,31 @@ const withBaseName = makePrefixer("saltList");
 const defaultEmptyMessage = "No data to display";
 
 export interface ListNextProps extends HTMLAttributes<HTMLUListElement> {
+  /**
+   * If true, all items in list will be disabled.
+   */
   disabled?: boolean;
+  /**
+   * Use to override the default empty message.
+   */
   emptyMessage?: string;
+  /**
+   * The component used to render a ListItem instead of the default. This must itself render a ListItem,
```

Comment:
```suggestion
   * The component used to render a list item instead of the default inbuilt LisItemNext component. This must itself render a ListItem,
```